File: android/app/src/main/java/com/lightningkite/kwifttemplate/api/APIInterface.shared.kt
package com.lightningkite.kwifttemplate.api

import com.lightningkite.kwift.escaping
import com.lightningkite.kwifttemplate.models.Post

interface APIInterface {
    //https://jsonplaceholder.typicode.com/posts
    fun getExamplePosts(
        onResult: @escaping() (Int, List<Post>?, String?) -> Unit
    )
}

File: ios/Kwift Template/com/lightningkite/kwifttemplate/api/APIInterface.shared.swift
//Package: com.lightningkite.kwifttemplate.api
//Converted using Kwift2

import Foundation
import Kwift
import KwiftMaps



public protocol APIInterface {
    
    func getExamplePosts(onResult: @escaping (Int32, Array<Post>?, String?) -> Void) -> Void
}
 
